include/ppx/geometry.h

```diff
@@ -169,68 +169,64 @@ class Geometry
         uint32_t    GetElementCount() const;
         uint32_t    GetDataSize() const;
 
-        // Append() will automatically adjust mOffset, but it may need to be
-        // manually adjusted if Overwrite() is used
-        void SetOffset(uint32_t offset) { mOffset = offset; }
-
         // Trusts that calling code is well behaved :)
         //
+        // Write(count, pValues...) is used to write a chunk of data at specified offset
+        // If called with offset=-1, will write at the end and adjust mOffset accordingly
```

Comment:
probably better with "will `append` at the end and adjust mOffset accordingly" so that it is clearer that it is appending not overwriting the value